packages/react/src/components/variants-provider.component.tsx

```diff
@@ -0,0 +1,159 @@
+import * as React from 'react'
+import {
+  builder,
+  Builder,
+  BuilderContent,
+  BuilderContentVariation
+} from '@builder.io/sdk'
+
+function getData(content: BuilderContentVariation) {
+  if (typeof content?.data === 'undefined') {
+    return undefined
+  }
+
+  const newData: any = {
+    ...content.data,
+    blocks: content.data.blocks || JSON.parse(content.data.blocksString)
+  }
+
+  delete newData.blocksString
+  return newData
+}
+
+const variantsScript = (variantsString: string, contentId: string) => `
+(function() {
+  if (typeof document.createElement("template").content === 'undefined') {
+    // IE 11
+    return ;
+  }
```

Comment:
maybe in this case just delete the templates and always show the default? 

otherwise if `template` means nothing they'd see every variation until JS loads right? and then the hydration from react could be broken here too